src/screens/LogIn.js

```diff
@@ -3,12 +3,30 @@ import React, { Component } from "react";
 class LogIn extends Component {
    render() {
       return (
-         <div>
-            <h2>HELLO</h2>
-            <p>Yup LogIn page loads</p>
+         <div className="logInFormContainer">
+            <input className="userInput" ref="firstNameText" type="text" size="30" placeholder="First Name" />
+            <br />
+            <input className="userInput" ref="lastNameText" type="text" size="30" placeholder="Last Name" />
+            <br />
+            <select className="categoryDropdown" ref='categoryType' >
```

Comment:
We don't need the list of acceptable schools since we aren't going to use them for anything

Reply:
the login page doesn't function yet, and isn't part of this sprint. I removed that stuff though.

Reply:
That's fine, just need to make sure we don't create more tech debt when making stories. And we shouldn't have work outside of the scope of the story unless it is laying the groundwork and will only be added to later